utility/esp_idf/RF24_arch_config.h

```diff
@@ -18,6 +18,7 @@
 #define _BV(x) (1 << (x))
 #define _SPI   SPIClass
 #define RF24_SPI_PTR
+#define RF24_SPI_TRANSACTIONS
```

Comment:
This would actually go in spi.h (per Arduino core convention). I commented it out when trying different things...
https://github.com/nRF24/RF24/blob/1eb16fa236cae82512cabaa5a0e09c55e767f17e/utility/esp_idf/spi.h#L23

Reply:
Fixed it.